packages/ui-patterns/form/FormLayout2.tsx

```diff
@@ -0,0 +1,434 @@
+import { zodResolver } from '@hookform/resolvers/zod'
```

Comment:
I'm guessing this file was committed by mistake or it should be a story?

Reply:
it is a story. its imported into a story elsewhere.
confusing i know